docs/data/data-grid/export/ExcelCustomExport.js

```diff
@@ -0,0 +1,478 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import {
+  DataGridPro,
+  useGridApiContext,
+  useGridSelector,
+  gridFilteredDescendantCountLookupSelector,
+  GridEvents,
+  GridToolbar,
+} from '@mui/x-data-grid-pro';
+import { styled } from '@mui/material/styles';
+import { Box, Button } from '@mui/material';
+
+const Link = styled('a')({
+  textOverflow: 'ellipsis',
+  whiteSpace: 'nowrap',
+  overflow: 'hidden',
+  color: 'inherit',
+});
+
+const isNavigationKey = (key) =>
+  key === 'Home' ||
+  key === 'End' ||
+  key.indexOf('Arrow') === 0 ||
+  key.indexOf('Page') === 0 ||
+  key === ' ';
+
+const CustomGridTreeDataGroupingCell = (props) => {
```

Comment:
Does the custom grouping cell bring any value here ?
If not could we remove it to keep the demo focused on the Excel export ?

Reply:
Yes, it does not bring that much value